api/src/lib.rs

```diff
@@ -1,7 +1,7 @@
-use std::collections::HashSet;
-
+use bundle::types::Test;
 use context::repo::RepoUrlParts;
 use serde::{Deserialize, Serialize};
+use std::collections::HashSet; // Add this import
```

Comment:
Can we undo the change to `std::collections::HashSet` here? It was more nicely organized before